### PR TITLE
Fixes #34227 - Properly reference top-level Rails module

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Review by ofedoren — COMMENTED** [review]
Thanks, @adamruzicka, LGTM. Although it should be quite safe to just merge, is it worth to review more places, e.g. https://github.com/theforeman/foreman-tasks/blob/ef88735b5f0026cbb4f421cbf1492bf02b9d0e5c/lib/foreman_tasks/dynflow/configuration.rb#L25 (wouldn't it look for `Rails` under `Dynflow` namespace there as well?)

**Comment by adamruzicka** [comment]
It seems to resolve correctly in all the other places, although I wouldn't mind being explicit (and safe) everywhere

**Comment by adamruzicka** [comment]
Replaced all occurences of `Rails` with `::Rails` just to be safe

**Review by ezr-ondrej — APPROVED** [review]
Thanks @adamruzicka ! :+1:

File: lib/foreman_tasks/dynflow/configuration.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-require File.expand_path('lib/foreman/dynflow/configuration', Rails.root)
+require File.expand_path('lib/foreman/dynflow/configuration', ::Rails.root)
```
Comment (ezr-ondrej) [inline lib/foreman_tasks/dynflow/configuration.rb:1]: FYI This actually should be `Rails.root.join('lib', 'foreman' ...)`. But it's unrelated to this PR :)

**Pull request merged**